modules/power-monitoring-deploying-kepler-redfish.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+
+// * power_monitoring/installing-power-monitoring.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="power-monitoring-deploying-kepler-redfish_{context}"]
+= Deploying {PM-kepler} using Redfish API
+
+You can use the Redfish API as the source to deploy {PM-kepler}. Redfish API increases the accuracy and efficiency of {PM-kepler} under certain circumstances.
+
+.Prerequisites
+* You have access to the {product-title} web console.
+* You are logged in as a user with the `cluster-admin` role.
+* You have installed the {PM-operator}.
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, go to *Operators* -> *Installed Operators*.
+
+. Click *{PM-title-c}* from the *Installed Operators* list and go to the *{PM-kepler}* tab.
+
+. Click *Create {PM-kepler}*.
+
+. Create a {PM-kepler} instance with `.spec.exporter.redfish` specifying the `secretRef`. `secretRef` refers to the name of secret which contains credentials to initialize `RedfishClient`.
```

Comment:
We need to mention that, clicking create will prompt power-monitoring operator to deploy kepler which  in turn creates the `namespace` `openshift-power-montoring` where the secret can be created.

Reply:
Made some changes to the steps following your comment. PTAL. :)